vic/drivers/shared_all/src/compute_derived_state_vars.c

```diff
@@ -195,50 +188,42 @@ compute_derived_state_vars(all_vars_struct *all_vars,
                         }
                     }
 
-                    /* initialize layer moistures and ice contents */
-                    for (lidx = 0; lidx < options.Nlayer; lidx++) {
-                        cell[veg][band].layer[lidx].moist =
-                            moist[veg][band][lidx];
-                        for (frost_area = 0;
-                             frost_area < options.Nfrost;
-                             frost_area++) {
-                            cell[veg][band].layer[lidx].ice[frost_area] =
-                                ice[veg][band][lidx][frost_area];
-                        }
-                    }
+                    /* calculate soil layer temperatures  */
                     if (options.QUICK_FLUX) {
                         ErrorFlag =
-                            estimate_layer_ice_content_quick_flux(
+                            estimate_layer_temperature_quick_flux(
                                 cell[veg][band].layer,
                                 soil_con->depth, soil_con->dp,
-                                energy[
-                                    veg][band].T[0], energy[veg][band].T[1],
-                                soil_con->avg_temp, soil_con->max_moist,
-                                soil_con->expt, soil_con->bubble,
-                                soil_con->frost_fract, soil_con->frost_slope,
-                                soil_con->FS_ACTIVE);
+                                energy[veg][band].T[0],
+                                energy[veg][band].T[1],
+                                soil_con->avg_temp);
                         if (ErrorFlag == ERROR) {
                             log_err("Error in "
-                                    "estimate_layer_ice_content_quick_flux");
+                                    "estimate_layer_temperature_quick_flux");
```

Comment:
Change to: 

```C
    log_err("Error in estimate_layer_temperature_quick_flux");
```

unless this is an `uncrustify` result in which case just leave it.